### DISCUSSION TITLE
Flatpak Package

### DISCUSSION THREAD

**Discussion by istori1** [LizardByte/community#38] [Sunshine] [answered]
Manifest for Flatpak

I am not sure if this license would need to be applied to it
https://github.com/flathub/flathub/blob/master/COPYING

[com.github.sunshinestream.sunshine.yml.txt](https://github.com/SunshineStream/Sunshine/files/8913958/com.github.sunshinestream.sunshine.yml.txt)

**Comment by ReenigneArcher** [comment]
This might be useful: https://github.com/bilelmoussaoui/flatpak-github-actions

**Comment by istori1** [comment, marked as answer]
Here are the official statements about licensing: https://github.com/flathub/flathub/wiki/App-Requirements#license
"All content hosted on Flathub must allow legal redistribution, and the license must be correctly specified in the app's appdata file."

In the Appstream section https://github.com/flathub/flathub/wiki/App-Requirements#appstream it mentions this license for the metadata: https://creativecommons.org/choose/zero/

**Reply by ReenigneArcher** [reply]
Are all the dependencies built in the flatpak yml redistributable? If so, we're okay on that part.

I'll need to look into Appstream a bit more... seems like we need to include a specific metadata file in the flatpak repo?

Another thing I notice on that page is about the desktop file... will the existing desktop file work? It's used for the AppImage right now and configured with cmake to ensure version and a few variables are populated at build time.

Can you link a good example flatpak repo that I can use for reference? I work better with examples.

**Reply by istori1** [reply]
I believe I have covered all dependencies needed for sunshine on the manifest. They are compiled from source except for the CUDA libraries.

Based on the following information https://github.com/flathub/flathub/wiki/App-Requirements#desktop-integration we may not need a desktop file.

Here is an example on an existing flatpak. https://github.com/flathub/org.ryujinx.Ryujinx
